src/containers/header/header.jsx

```diff
@@ -168,6 +169,7 @@ export default compose(
         hideMenu: () => toggleMenu(false),
 
         height: isMenuOpen && isPhone() ? '100vh' : 'auto',
+        isPhoneResolution: isPhone(),
```

Comment:
This value won't change on `resize` event. This case is applicable to desktop users who decided to resize their browser window only but still needed.
I think should implement all logic that use this variable in CSS.